src/deprecated.jl

```diff
@@ -235,3 +235,5 @@ import Base: names
 @deprecate h5g_open(loc_id, pathname) h5g_open(loc_id, pathname, HDF5.H5P_DEFAULT) false
 @deprecate h5l_exists(loc_id, pathname) h5l_exists(loc_id, pathname, HDF5.H5P_DEFAULT) false
 @deprecate h5o_open(loc_id, name) h5o_open(loc_id, name, HDF5.H5P_DEFAULT) false
+
+@deprecate writearray(obj::Attribute, type_id, x) a_write(obj, type_id, x) false
```

Comment:
I don't mind adding this, but I really hope no one was using this internal call.

Reply:
JLD and MAT both do: https://juliahub.com/ui/CodeSearch?q=writearray&u=all&t=all